### DISCUSSION TITLE
change multiple selected segment order

### DISCUSSION THREAD

**Discussion by leoguan888** [mifi/lossless-cut#2227] [Q&A]
My needs is to reorder lots of video segments. In current version, I can only move them individually using the window on the right, which displays limited content.
I wish it can able to move all selected segments at once.

**Comment by mifi** [comment]
What do you mean by «move»? There’s a main menu function to shift all selected segments times

**Reply by leoguan888** [reply]
For example, rearrange video segments 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 into the order 1, 3, 5, 7, 9, 2, 4, 6, 8, 10. I wish it can select 1, 3, 5, 7, 9 segment to move forward at once. Currently I can only arrange the segments one by one which is inconvenient when I have like 50 segments to rearrange.

**Reply by mifi** [reply]
Ah I see. It's not super trivial to implement, but I've added your suggestion to #392, thanks
